packages/venia-concept/src/components/CategoryList/categoryList.js

```diff
@@ -0,0 +1,105 @@
+import { Component, createElement } from 'react';
+import { string, number, shape } from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import classify from 'src/classify';
+import defaultClasses from './categoryList.css';
+
+// TODO: get only active categories from graphql when it is ready
+const categoryListQuery = gql`
+    query category($id: Int!) {
+        category(id: $id) {
+            children {
+                name
+                url_key
+                url_path
+                image
+            }
+        }
+    }
+`;
+
+// TODO: get baseUrl from graphql when it is ready
+const baseUrl = 'https://magento-venia.local.pwadev:8000';
+
+// TODO: get mediaUrl from graphql when it is ready
+const mediaUrl = 'https://magento-venia.local.pwadev:8000/pub/media';
+
+// TODO: get categoryUrlSuffix from graphql when it is ready
+const categoryUrlSuffix = '.html';
+
+class CategoryList extends Component {
+    static propTypes = {
+        id: number,
+        title: string,
+        classes: shape({
+            root: string,
+            header: string,
+            content: string,
+            title: string,
+            item: string,
+            imageWrapper: string,
+            image: string,
+            name: string
+        })
+    };
+
+    static defaultProps = {
+        id: 2
```

Comment:
This component probably shouldn't have a default ID. I'm guessing you made this for easy testing, but it should be removed.

Reply:
Idea was to have preset of the default magento category.
"Root Catalog" - id="1"
"Default Category" - id="2"